.changes/v2.24.0/665-features.md

```diff
@@ -0,0 +1,6 @@
+* Added method `DefinedEntity.Refresh` to reload RDE state [GH-665]
```

Comment:
Suggestion:
```
* Added types `SolutionLandingZone` and `types.SolutionLandingZone` for Solution Add-on Landing Zone configuration [GH-665]
* Added method `DefinedEntity.Refresh` to reload RDE state [GH-665]
* Added `VDCClient` methods `CreateSolutionLandingZone`, `GetAllSolutionLandingZones`,
  `GetExactlyOneSolutionLandingZone`, `GetSolutionLandingZoneById` for handling Solution Landing Zones [GH-665]
* Added  `SolutionLandingZone` methods `Refresh`, `Id`, `Update`,
  `Delete` to help handling of Solution Landing Zones [GH-665]
```

Reply:
Agreed, changed (only changed `Id` to `RdeId` as per the other comment)